extensions/2.0/Vendor/EXT_primitive_voxels/README.md

```diff
@@ -65,13 +69,19 @@ The `dimensions` property of the extension specifies the voxel grid dimensions:
 
 Dimensions must be nonzero. Elements are laid out in memory first-axis-contiguous, e.g. for boxes, `x` data is contiguous (up to stride).
 
+The `bounds` property describes how the voxel is clipped before rendering in voxel space. `bounds.minimum` and `bounds.maximum` specify a "rectangular" region of the voxel in the appropriate
```

Comment:
I think there's a better word than `clipped` here since the data is being scrunched into a smaller space, not discarded.